flags/14.0/vttablet.txt

```diff
@@ -500,23 +354,24 @@ max_rate_approach_threshold: 0.9
       --vreplication_healthcheck_timeout duration                        healthcheck retry delay (default 1m0s)
       --vreplication_healthcheck_topology_refresh duration               refresh interval for re-reading the topology (default 30s)
       --vreplication_heartbeat_update_interval int                       Frequency (in seconds, default 1, max 60) at which the time_updated column of a vreplication stream when idling (default 1)
-      --vreplication_max_time_to_retry_on_error duration                 stop automatically retrying when we've had consecutive failures with the same error for this long after the first occurrence (default 15m0s)
+      --vreplication_max_time_to_retry_on_error duration                 stop automatically retrying when we've had consecutive failures with the same error for this long after the first occurrence
```

Comment:
⚠️  We are missing the default here. (default is 0=>keep trying).

Reply:
In pflag, when the default is the zero value for the type, it doesnt bother printing. No todo here